client/src/hooks/graphql/use-with-setup.tsx

```diff
@@ -96,6 +99,7 @@ export function useWithSetup(
     startTask: startTraining,
     clearError: clearTrainingError,
   } = useWithTraining();
+  const config = useWithConfig();
```

Comment:
I think we probably want `useWithSetup` to bail out immediately if config is not loaded? If we can't do that, we need to change how we handle the case of "config not loaded" below